apps/server/package.json

```diff
@@ -9,15 +9,16 @@
   "devDependencies": {
     "@types/jest": "^29.5.1",
     "@types/node": "^18.16.7",
-    "@wundergraph/wunderctl": "^0.149.0",
+    "@wundergraph/wunderctl": "^0.159.0",
```

Comment:
@0xJem is there any reason why you install `wunderctl` separately?

Reply:
No, because you don't need to keep it in sync manually. The SDK ships the compatible version. Remove it and `wunderctl` should still work:smile:

Reply:
Thanks, fixed in #37